instant/flags.vim

```diff
@@ -104,3 +104,12 @@ call s:plugin.Flag('shfmt_options', ['-i', '2', '-sr', '-ci'])
 ""
 " The path to the shfmt executable.
 call s:plugin.Flag('shfmt_executable', 'shfmt')
+
+""
+" Command line arguments to to feed prettier. Either a list or callable that
```

Comment:
Need to run vimdoc over the directory (see https://github.com/google/vimdoc) so the doc file picks up this edit.

Reply:
Done